### ISSUE TITLE
Add job metrics for absolute resource consumption

### ISSUE THREAD

**Issue by damzog** [bosh-prometheus/bosh_exporter#24]
As an operator of bosh based deployments in order to bill to my customer and do capacity planning I would like to have metrics on absolute resources allocated (cpu, ram, disk) for each deployment. Basically it should be a sum of all ram, cpu, disk values for each instance_group instance as defined in manifest/cloud config. 

Please comment if you find this requirement reasonable to be placed in prometheus. I am considering to implement it myself.

**Comment by frodenas** [comment]
How do you plan to get that information? Some CPIs does not expose that information, for example, AWS only supports an `instance_type` that does not provide the resources information.

**Comment by damzog** [comment]
Good point. I did not realize that at first sight as I am mostly working with vsphere. One idea I have is to expose a metric like 
```vm_type_count{deployment="some-deployment", vm_type="small", az="z1"} integer```
that can be later expanded to actual resource usage. Probably the same would be needed for disk_type
Other solution would be to include vm_type sizing in the bosh_exporter config which is of course not so nice as it has its definition elsewhere.
Another idea is to use node exporter but we did not install node exporter on every payload vm and I am not sure if this scales well to hundreds of vms.
